.github/workflows/build-docs.yaml

```diff
@@ -7,9 +7,11 @@ on:
       - "!main"
     tags-ignore:
       - "*"
+  pull_request:
 
 jobs:
   build-docs:
+    if: github.event_name != 'pull_request' || github.event.pull_request.head.repo.fork
```

Comment:
It'd be cool if we could put the `if` inside of the `pull_request` trigger above, but GitHub actions don't support that. Only once we're in the `job` definition can we specify a condition to run, so we need to check which event triggered it in addition to if it is a forked repo.

Reply:
That would be nice! Maybe one day 🤞

Reply:
in hindsight, this PR comment should probably be a comment in the file itself. I'm going to add that.